react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/BasePaywallViewManager.kt

```diff
@@ -25,16 +34,31 @@ internal abstract class BasePaywallViewManager<T : View> : SimpleViewManager<T>(
             .build()
     }
 
-    @ReactProp(name = "options")
+    @OptIn(ExperimentalPreviewRevenueCatUIPurchasesAPI::class)
+    abstract fun setFontFamily(view: T, customFontProvider: CustomFontProvider)
+
+    @ReactProp(name = PROP_OPTIONS)
     fun setOptions(view: T, options: ReadableMap?) {
         options?.let { props ->
-            if (props.hasKey("offering")) {
-                props.getDynamic("offering").asMap()?.let { offeringMap ->
-                    if (offeringMap.hasKey("identifier") && !offeringMap.isNull("identifier")) {
-                        setOfferingId(view, offeringMap.getString("identifier")!!)
-                    }
-                }
+            setOfferingIdProp(view, props)
+            setFontFamilyProp(view, props)
+        }
+    }
+
+    private fun setOfferingIdProp(view: T, props: ReadableMap?) {
+        val offeringIdentifier = props?.getDynamic(OFFERING)?.asMap()?.getString(IDENTIFIER)
+        offeringIdentifier?.let {
+            setOfferingId(view, it)
+        }
+    }
+
+    @OptIn(ExperimentalPreviewRevenueCatUIPurchasesAPI::class)
```

Comment:
Is this needed? I just noticed `setOfferingIdProp` doesn't have it

Reply:
It's needed since `CustomFontProvider` is marked with the experimental annotation